code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -75,7 +75,8 @@
 				floor_only = FALSE
 			else
 				qdel(C)
-		T.color = initial(T.color)
+		if(!istype(T,/turf/simulated/floor/light))
```

Comment:
```suggestion
		if(!istype(T, /turf/simulated/floor/light))
```
This also feels kinda snowflakey but dont know another way around that

Reply:
The only way around it would be to change the way light tiles work completely, so that they don't use the T.color variable. Or change colorful reagent and any other color changers so that they don't use T.color.